x-pack/test/functional/apps/transform/editing.ts

```diff
@@ -0,0 +1,154 @@
+        await transform.navigation.navigateTo();
+        await transform.management.assertTransformListPageExists();
+      });
+
+      it('should display the transforms table', async () => {
+        await transform.management.assertTransformsTableExists();
+      });
+
+      it('should display the original transform in the transform list', async () => {
+        await transform.table.refreshTransformList();
+        await transform.table.filterWithSearchString(transformConfig.id);
+        const rows = await transform.table.parseTransformTable();
+        expect(rows.filter((row) => row.id === transformConfig.id)).to.have.length(1);
+      });
+
+      it('should show the actions popover', async () => {
+        await transform.table.assertTransformRowActions(false, 'transformActionEdit');
+      });
+
+      it('should show the edit flyout', async () => {
+        await transform.editFlyout.assertTransfromEditFlyoutExists();
+      });
+
+      it('should update the transform description', async () => {
+        await transform.editFlyout.assertTransformEditFlyoutInputExists('Description');
+        await transform.editFlyout.assertTransformEditFlyoutInputValue(
+          'Description',
+          transformConfig?.description ?? ''
+        );
+        await transform.editFlyout.setTransformEditFlyoutInputValue(
+          'Description',
+          testData.transformDescription
+        );
+      });
+
+      it('should update the transform documents per second', async () => {
+        await transform.editFlyout.assertTransformEditFlyoutInputExists('DocsPerSecond');
+        await transform.editFlyout.assertTransformEditFlyoutInputValue('DocsPerSecond', '');
+        await transform.editFlyout.setTransformEditFlyoutInputValue(
+          'DocsPerSecond',
+          testData.transformDocsPerSecond
+        );
+      });
+
+      it('should update the transform frequency', async () => {
+        await transform.editFlyout.assertTransformEditFlyoutInputExists('Frequency');
+        await transform.editFlyout.assertTransformEditFlyoutInputValue('Frequency', '');
+        await transform.editFlyout.setTransformEditFlyoutInputValue(
+          'Frequency',
+          testData.transformFrequency
+        );
+      });
+
+      it('should update the transform', async () => {
+        await transform.editFlyout.updateTransform();
+      });
+
+      it('should display the transforms table', async () => {
+        await transform.management.assertTransformsTableExists();
+      });
+
+      it('should display the updated transform in the transform list', async () => {
+        await transform.table.refreshTransformList();
+        await transform.table.filterWithSearchString(transformConfig.id);
+        const rows = await transform.table.parseTransformTable();
+        expect(rows.filter((row) => row.id === transformConfig.id)).to.have.length(1);
+      });
+
+      it('should display details for the update job in the job list', async () => {
```

Comment:
It isn't clear to me what this test is for from the description, which refers to `job` and `job list`. Looks like it is checking each of the fields in the row for the updated transform. In which case, something like `should display the updated fields for the transform in the transforms list` ?

Reply:
Updated in a037c8d. It now says `should display the updated job in the job list row cells`.